### DISCUSSION TITLE
请问Higress占用443端口后如何配置进行访问前端dist文件

### DISCUSSION THREAD

**Discussion by SimleXin** [alibaba/higress#878] [Q&A]
前端构建的dist包 被我放在与higress同机器上
80与443端口被higress占用 同时我也不想再使用nginx了
如何配置higress可以使请求访问到前端dist包或服务器的静态资源文件夹？

**Comment by CH3CHO** [comment]
没太理解你的问题，能详细介绍一下上下文么，从机器上完全没有Higress到出问题的整个流程？

**Reply by SimleXin** [reply]
就是以往使用gateway开发   服务器上使用nginx当做网关   nginx可以同时反向代理gateway的请求和 前端dist包的请求 

nginx同样也是用80 和443端口  使用higress的话  不就是不能使用nginx了么    higress如何让请求映射到前端的dist包上

**Reply by SimleXin** [reply]
使用higress只能接收API的请求   比如我的请求访问服务器上的静态资源    
https://xxx.xxx/static/image.jpg
通过nginx可以直接返回/home/image.jpg

使用higress该如何处理呀？

**Reply by SimleXin** [reply]
我们这种微型企业 通常只有一台服务器 跑全部付的服务  没有额外的单独资源服务器

**Reply by CH3CHO** [reply]
目前 Higress 应该还不支持静态资源路由功能。请 @johnlanni 看一下。

**Reply by johnlanni** [reply]
是的这个暂时不支持

**Reply by SimleXin** [reply]
加油大佬们 早日更新上  我在用nginx一阵
